src/main/no/nav/pto/veilarbfilter/Application.kt

```diff
@@ -52,11 +52,11 @@ fun main(configuration: Configuration) {
     Runtime.getRuntime().addShutdownHook(Thread {
         applicationState.initialized = false
         applicationServer.stop(5, 5)
-        cleanUpVeilederGrupper.stop()
+        //cleanUpVeilederGrupper.stop()
```

Comment:
skal kommentarer være med?

Reply:
That part was not functioning (request to get veilederer på enheten was returning 500 code and no data), so I commented it out until we fix it.

Reply:
The cleanup part will be returned very soon.